Sim/SimPapas/src/PapasAlg.cpp

```diff
@@ -0,0 +1,62 @@
+#include "PapasAlg.h"
+
+#include "SimPapas/IPapasTool.h"
+#include "papas/utility/PDebug.h"
+#include "papas/utility/TRandom.h"
+
+#include <iostream>
+DECLARE_COMPONENT(PapasAlg)
+
+PapasAlg::PapasAlg(const std::string& name, ISvcLocator* svcLoc) : GaudiAlgorithm(name, svcLoc) {
+  declareProperty("tools", m_toolNames);
+  papas::PDebug::File("papaslog.log");
```

Comment:
There are two things. 
I have made an issue for Papas to (hopefully) directly support Gaudi Logging #221. 
I have made the physics Debug output from Papas configurable.